### PR TITLE
Removing sexually related objects

### PR DESCRIPTION
**Pull request by KeplerWasTaken**
## About the PR

Removes a lot of sex pins and some sort of maid-catgirl thing

**Media**

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- remove: Sexual stuff; this is not an ERP server

### CODE REVIEW

**Comment by MasterSneed** [comment]
We have already added middle eastern localization to hippie

**Comment by Chudjak** [comment]
Some of these I agree with, some of these I do not.
Nyanify is the same as catbeasting, maid is funny and no cumbrains will dissuade me of that and cat ears should remain in the code simply due to the fact that they always were there. We aren't an ERP server, if anyone uses these to ERP they just get banned.
Keep it to only the pins.

**Comment by Chudjak** [comment]
Not as a syndicate item.
Add the stuff I told you back.

**Comment by KeplerWasTaken** [comment]
okay
I will add in the smites back

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 1]
```diff
@@ -845,6 +845,19 @@
...
+ 
+ 
+ 
+- type: listing
+  id: UplinkCatEars
+  name: uplink-cat-ears-name
+  description: uplink-cat-ears-desc
+  productEntity: ClothingHeadHatCatEarsCursed
+  cost:
+    Telecrystal: 26
```
Comment (Chudjak) [inline Resources/Prototypes/Catalog/uplink_catalog.yml]: wtf?

**Comment by KeplerWasTaken** [comment]
okay the cat shit has been re-added, still have yet to test

**Comment by Chudjak** [comment]
it's been a week, you tested it yet?

**Comment by KeplerWasTaken** [comment]
ye ill test it now. testing ss13 crap always been a pain, even worse with the ugly ass UI of ss14. Dark mode is retarded

**Comment by KeplerWasTaken** [comment]
has some bugs working on it now.

**Comment by KeplerWasTaken** [comment]
okay bugs gone game launches bye sex

**Comment by BarginMart** [comment]
Good Riddance

**Pull request merged**